### PR TITLE
Bug 1832762: updates Dockerfile.rhel to golang 1.13 and ocp:4.4

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: images/Dockerfile.rhel [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
```
Comment (tnozicka) [inline images/Dockerfile.rhel:1]: you need to follow up with updating the CI config

Comment (p0lyn0mial) [reply images/Dockerfile.rhel:1]: done - https://github.com/openshift/release/pull/8905

**Pull request merged**